components/namespace-lister/base/metrics/monitor.yaml

```diff
@@ -18,9 +18,9 @@ spec:
         ca:
           secret:
             key: tls.crt
-            name: namespace-lister-tls
+            name: namespace-lister-metrics-tls
```

Comment:
hmmm, we don't have a method for using different keys for both the api server and the metrics server.  Why change this?

Reply:
right, so we need to rely on the cert generated for the other service... that's a bit ugly

Reply:
should be fixed by 3f25ba1e01d24fddd928f26a1e4a330035050787